.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
+
+**What kind of change does this PR introduce?**
+
+<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->
+
+**If relevant, did you add tests for your changes?**
+
+
+**If relevant, link to documentation update:**
```

Comment:
not sure how many times'll really get this

Reply:
oh right! we only have one documentation. this is not a valid one (removed)